### DISCUSSION TITLE
Unified alerting -Need a way to restrict contexts

### DISCUSSION THREAD

**Discussion by nerdyslacker** [grafana/grafana#45908] [Alerting]
I've configured unified alerts and alertmanager datasource in openshift cluster and I am able to see all alerts from my 4 contexts. 
I need to restrict alerts to be shown only from one context. Is there any way to do that ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
